iothub/device/src/ITransportSettings.cs

```diff
@@ -19,6 +19,6 @@ public interface ITransportSettings
         /// <summary>
         /// The default receive timeout.
         /// </summary>
-        TimeSpan DefaultReceiveTimeout { get; }
```

Comment:
Wouldn't this qualify as a breaking API change?

Reply:
Yes, I didn't mean for that change to happen. Reverted.